interface/ceed-operator.c

```diff
@@ -106,6 +107,7 @@ int CeedCompositeOperatorCreate(Ceed ceed, CeedOperator *op) {
   (*op)->ceed = ceed;
   ceed->refcount++;
   (*op)->composite = true;
+  (*op)->numelements = -2;
```

Comment:
What does `-2` mean?

Reply:
I can add a comment.
Old schema
0 - unset
1+ - set
New schema
-2 - composite operator
-1 - unset
0+ - set

I'm using -2 as a nonzero value that isn't -1 to simplify the errorchecking logic.

Reply:
Please document this schema. It looks hazardous in this PR because you only check `numelements != -1` instead of `numelements >= 0` (though perhaps it is intended that those places be unreachable for composite operators).

Reply:
I could also go real crazy and set a boolian flag if a restriction has been added and then only accept values >=0. It would probably be clearer.